website/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "website",
+  "type": "module",
+  "version": "0.0.1",
```

Let's make it private.
```suggestion
  "version": "0.0.1",
  "private": true,
```